ethtool.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: ethtool
+  version: 6.2
+  epoch: 0
+  description: Utility for controlling network drivers and hardware
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: GPL-2.0-only
```

Comment:
```suggestion
  copyright:
    - license: GPL-2.0-only
```

Reply:
Since #709 these blocks aren't used elsewhere. It's not a problem if they're there, they're just ignored, but it'd be nice to stay consistent.

Reply:
lemme remove those